include/alpaka/rand/RandDefault.hpp

```diff
@@ -0,0 +1,198 @@
+                    };
+                } // namespace detail
+
+                //! The GPU random number normal distribution.
+                template<typename T>
+                class UniformUint
+                {
+                public:
+                    UniformUint() = default;
+
+                    template<typename TEngine>
+                    ALPAKA_FN_HOST_ACC auto operator()(TEngine& engine) -> T
+                    {
+                        using BitsT = typename TEngine::result_type;
+                        T ret = 0;
+                        constexpr auto N = sizeof(T) / sizeof(BitsT);
+                        for(unsigned int a = 0; a < N; ++a)
+                        {
+                            ret ^= (static_cast<T>(engine()) << (sizeof(BitsT) * 8 * a));
+                        }
+                        return ret;
+                    }
+                };
+
+                //! The GPU random number uniform distribution.
+                template<typename T>
+                class UniformReal
+                {
+                    using BitsT = typename detail::BitsType<T>::type;
+
+                public:
+                    UniformReal() = default;
+
+                    template<typename TEngine>
+                    ALPAKA_FN_HOST_ACC auto operator()(TEngine& engine) -> T
+                    {
+                        constexpr BitsT limit = static_cast<BitsT>(1) << std::numeric_limits<T>::digits;
+                        const BitsT b = UniformUint<BitsT>()(engine);
+                        const auto ret = static_cast<T>(b & (limit - 1)) / limit;
+                        return ret;
+                    }
+                };
+
+                //! The GPU random number normal distribution.
+                template<typename Acc, typename T>
+                class NormalReal
+                {
+                    const Acc& m_acc;
+                    T m_buf = static_cast<T>(NAN);
+
+                public:
+                    NormalReal(const Acc& acc) : m_acc(acc)
+                    {
+                    }
+
+                    template<typename TEngine>
+                    ALPAKA_FN_HOST_ACC auto operator()(TEngine& engine) -> T
+                    {
+                        constexpr T sigma = 1., mu = 0.;
+                        if(std::isnan(m_buf))
+                        {
+                            UniformReal<T> uni;
+
+                            T u1, u2;
+                            do
+                            {
+                                u1 = uni(engine);
+                                u2 = uni(engine);
+                            } while(u1 <= std::numeric_limits<T>::epsilon());
+
+                            // compute z0 and z1
+                            const T mag = sigma * math::sqrt(m_acc, static_cast<T>(-2.) * math::log(m_acc, u1));
+                            constexpr T TwoPi = static_cast<T>(2. * M_PI);
```

Comment:
Why is `TwoPi` starting with upper case latter and all other variables with lower?

Reply:
Fixed.